docs/docs/general/07-Troubleshooting/deprecated-endpoints.md

```diff
@@ -0,0 +1,15 @@
+# Deprecated Endpoints
+
+The askui library is sending requests with screenshots and/or instructions
+to endpoints of our inference API to get annotations and commands.
+We are starting to use our new `inference` endpoint for this kind of requests.
+Earlier versions (`v0.3.2` and prior) are using the `predict-comand` and `annotate` endpoint
+of the API.
+The old endpoints are still working but we are not planning to maintain them.
+Both endpoints will be deprecated in the future.
+
+Please upgrade your askui version to use our new endpoint if you are using askui `v0.3.2` and earlier:
```

Comment:
I don't like that you explained too much to the user about our current problem. and you made this chapter too specific for the current situation. 
from my perspective, this should be generic ( it only explains that if you get this warning, you may need to upgrade in the near future with this command. )  
would like to have the opinion from the others @JohannesDienst-askui , @adi-wan-askui , @programminx-askui

Reply:
good point. going for something shorter which is more generic. Also less changes in the future with this way

Reply:
A generic explanation is fine in my opinion